docs/en_US/builtin_assessor.rst

```diff
@@ -1,6 +1,16 @@
 Builtin-Assessors
 =================
 
+In order to save our computing resources, NNI supports an early stop policy and creates **Assessor** to finish this job.
+
+Assessor receives the intermediate result from Trial and decides whether the Trial should be killed by specific algorithm. Once the Trial experiment meets the early stop conditions(which means assessor is pessimistic about the final results), the assessor will kill the trial and the status of trial will be `"EARLY_STOPPED"`.
```

Comment:
> Assessor receives the intermediate result

Assessor receives the intermediate results

>the status of trial will be `"EARLY_STOPPED"`.

the status of trial will change to `"EARLY_STOPPED"`.